src/core/reducers/user.js

```diff
@@ -1,14 +1,21 @@
+/* @flow */
 import { LOG_OUT_USER } from 'core/constants';
 
 
 const USER_PROFILE_LOADED = 'USER_PROFILE_LOADED';
 
-export const initialState = {
+type Action = Object;
```

Comment:
TODO: have a better `Action` type. I find it difficult and a bit superfluous to maintain this type in general.